pkg/ansible/watches/watches.go

```diff
@@ -312,3 +297,18 @@ func getAnsibleVerbosity(gvk schema.GroupVersionKind, defValue int) int {
 	}
 	return ansibleVerbosity
 }
+
+// Returns value for MaxWorkers/Ansibleverbosity based on if envVar is set or a defvalue is used.
+func getIntegerEnvWithDefault(envVar string, defValue int) int {
+	val := defValue
+	if envVal, ok := os.LookupEnv(envVar); ok {
+		if i, err := strconv.Atoi(envVal); err != nil {
+			log.Info("could not parse environment variable as an integer; using default value", "envVar", envVar, "default", defValue)
+		} else {
+			val = i
+		}
+	} else if !ok {
+ 		log.Info("environment variable not set; using default value", "envVar", envVar, "default", defValue)
```

Comment:
Looks like the sanity test is complaining that these `log.Info` messages don't start with an upper case letter.

So we need:
`Could not parse environment variable...` and `Environment variable not set...`